lib/main.js

```diff
@@ -18,43 +18,54 @@ function HTMLUglify(config) {
 
   var salt = config.salt || 'use the force harry';
```

Comment:
Let's not be grody. 

https://github.com/ivanakimov/hashids.js/blob/master/lib/hashids.js#L289

`crypto.pseudoRandomBytes(ALPHABET.length).toString('utf8')`. Done.